src/VocabularyManager.UseCases/Services/OxfordDictionaryParser.cs

```diff
@@ -1,15 +1,18 @@
 ﻿using AngleSharp;
 using AngleSharp.Dom;
-using DictionaryParser.Exceptions;
-using DictionaryManager.Shared.Models.DTOs;
+using FluentValidation;
+using FluentValidation.Results;
+using VocabularyManager.UseCases.Exceptions;
+using VocabularyManager.UseCases.DTOs;
+using VocabularyManager.UseCases.Interfaces;
 
-namespace DictionaryParser.Services
+namespace VocabularyManager.UseCases.Services
```

Comment:
name of the folder it's `Services` and in that folder you have `Parser` and `Managers`.

Looks like this is folder for stuff. There is no clear purpose what this folder is and what it should contain